course_access_groups/serializers.py

```diff
@@ -1,13 +1,75 @@
-from rest_framework import serializers
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
 
+from rest_framework import serializers
+from openedx.core.lib.api.serializers import CourseKeyField
 from course_access_groups.models import (
-    CourseAccessGroup,
+    Group,
+    CourseGroup,
+    Membership,
+    Rule,
 )
 
 
 class GroupSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField()
```

Comment:
Interesting. Do you get an error if you don't explicitly add `id` as a class attribute?

Reply:
@johnbaldwin I think that it's possible to rewrite the ID, I want to prevent such use-case to avoid problematic edge-cases.

Reply:
nvm, we can live without those `id = serializers.ReadOnlyField()`